src/pages/LearnPage.tsx

```diff
@@ -70,7 +144,22 @@ export default function LearnPage() {
       <LessonGroup title="Intro Algo Design">
         <InlineSpacing spacing={40} />
 
-        <LessonButton>Brute Force</LessonButton>
+        <LessonButton
+          onClick={() =>
+            setLesson(
+              <LessonPage
+                blocks={[
+                  <ReadingBlock content={dp} />,
+                  <ReadingBlock content={markdown1} />,
+                  <ReadingBlock content={markdown2} />,
+                  <ReadingBlock content={markdown3} />,
```

Comment:
nit but can you rename `markdownX` to be a little more descriptive

Reply:
This is only placeholder content. There is no meaning to them and they will be removed once we start fetching the content from markdown files.